code/_compile_options.dm

```diff
@@ -70,4 +70,8 @@
 #endif
 
  #define SERVICE_CMD_PARAM_KEY "serviceCommsKey"
- 
\ No newline at end of file
+
+//Update this whenever the db schema changes 

```

Comment:
No, this is not a currently preserved file, I don't even know what that key is. 

@Cyberboss explain?